assets/dev/js/editor/utils/notifications.js

```diff
@@ -93,6 +95,24 @@ module.exports = elementorModules.Module.extend( {
 		return toast.show();
 	},
 
+	positionToWindow( toast ) {
+		const positionOf = toast.getSettings( 'position' ).of.replace( '#', '' );
+
+		if ( document.getElementById( positionOf ) ) {
+			return;
+		}
+
+		const position = {
+			...toast.getSettings( 'position' ),
+			my: 'right top',
+			at: 'right-10 top+42',
+			of: '',
+		};
```

Comment:
Why do you force the position to be this position?

Reply:
Because in the default settings it's centered to `#elementor-panel-content-wrapper` 10 pixels from the bottom.
Outside of the editor the panel does not exist, thus we need to align it to window, by changing `position.of` to an empty string. When it's center-bottom aligned at the admin screen it looks weird, thus in this case I move it to top right.

Reply:
NO, let's talk about it.